Rmd/quick_stats.Rmd

```diff
@@ -0,0 +1,267 @@
+                             class = "italy")
+do.call(timeline, timeline_italy_cls)
+```
+
+### State Level
+
+#### Confirmed cases
+
+The total number of confirmed cases in Texas is `r state  %>% dplyr::filter(stateAbb == "TX")  %>% dplyr::summarise(max = max(confirmed)) %>% dplyr::pull(max)`. That number has increased over the last several weeks, however it is likely that this number is underreported due to a lack of testing in the state of Texas.
+
+```{r}
+states <- state %>% 
+  dplyr::filter(stateAbb %in% c("TX", "CA", "WA", "FL", "NY"))
+
+gg <- ggplot(states,
+             aes(x = Date, y = confirmed, color = stateName)) +
+  geom_point() +
+  geom_line() +
+  scale_color_manual("States", values = as.vector(tpltheme::tpl_palettes$categorical)) +
+  labs(y = "Confirmed cases",
+       x = "Date",
+       caption = paste("Source:", usafacts_source),
+       title = "Confirmed cases over time") +
+  ylim(0, NA)
+
+print(gg)
+```
+
+#### Deaths
+
+The total number of deaths in Texas is `r states  %>% dplyr::filter(stateAbb == "TX")  %>% dplyr::summarise(max = max(deaths)) %>% dplyr::pull(max)`.
+
+```{r}
+
+gg <- ggplot(states,
+             aes(x = Date, y = deaths, color = stateName)) +
+  geom_point() +
+  geom_line() +
+  scale_color_manual("States", values = as.vector(tpltheme::tpl_palettes$categorical)) +
+  labs(y = "Deaths",
+       x = "Date",
+       caption = paste("Source:", usafacts_source),
+       title = "Confirmed deaths over time") +
+  ylim(0, NA)
+
+print(gg)
+```
+
+#### Testing
+
+The total number of tests in Texas is `r states  %>% dplyr::filter(stateAbb == "TX")  %>% dplyr::summarise(max = max(total_tests, na.rm = TRUE)) %>% dplyr::pull(max)`. On Monday March 16th, Governor Greg Abbott suggested that the testing capacity in Texas would ramp up substantially over the following week (March 16th thru March 22nd) to over 10,000 tests. However, testing has fallen short of this initial estimate. The governor promised that by the end of the week (March 22nd) "everyone who needs a COVID-19 test will be able to get a COVID-19 test", however this doesn't seem to be the case. Texas appears to be substantially lagging in ramping up it's testing capacity, which Governor Andrew Cuomo said was the first line of defense again COVID-19 (The Daily March 18th, 2020).
+
+```{r}
+gg <- ggplot(states,
+             aes(x = Date, y = total_tests, color = stateName)) +
+  geom_point() +
+  geom_line() +
+  scale_color_manual("States", values = as.vector(tpltheme::tpl_palettes$categorical)) +
+  labs(y = "Total # of tests",
+       x = "Date",
+       caption = paste("Source:", covid_tracking_source),
+       title = "Total # of tests over time") +
+  ylim(0, NA)
+
+print(gg)
+```
+
+
+#### Timelines
+
+##### Texas
+
+```{r}
+timeline_tx_cls <- structure(list(df = state,
+                                  state = "Texas",
+                                  title = "Confirmed cases in the state of {} over time",
```

Comment:
Right now, the title of each plot is not showing up. I fixed this on my system and will push soon.

Reply:
Great!